.github/workflows/pysat_rc.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Test with latest pysat RC
+
+on: [workflow_dispatch]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
```

Comment:
Not mac?

Reply:
not in the main jobs yet

Reply:
Might be worth adding a comment with a TODO?

Reply:
Added #101 with this and other things we are not currently testing in main due to fortran issues